### DISCUSSION TITLE
提问：能否提供有关代理设置的相关说明？

### DISCUSSION THREAD

**Discussion by BrightSphere** [lifegpc/bili#70] [Q&A] [answered]
尝试使用bili_1.4.2_windows10_x64版本start.exe --httpproxy 127.0.0.1:8000 方式设置代理，但实际并没有走代理。
请问正确的设置方式是什么呢？

**Comment by lifegpc** [comment, marked as answer]
httpsproxy

**Comment by lifegpc** [comment]
因为大多数连接都是https

**Comment by lifegpc** [comment]
如果使用aria2c的话，还需要设置ahttpsporxy。
或者也可以设置HTTP_PROXY和HTTPS_PROXY的环境变量。aria2c和python都会默认读取环境变量。
